core/src/main/resources/application.properties

```diff
@@ -12,6 +12,7 @@ spring.jpa.show-sql=false
 spring.jpa.properties.hibernate.session.events.log=false
 # logging.level.org.hibernate.SQL=DEBUG
 # logging.level.org.hibernate=DEBUG
+# logging.level.org.hibernate.type.descriptor.sql=TRACE
```

Comment:
This should not go to production, should it?

Reply:
It's commented. I left it in for convenience, like the above ones. Every time I needed to Google to find the log category.